convert_test.go

```diff
@@ -136,10 +144,19 @@ func Benchmark_ToString(b *testing.B) {
 		float32(3.14),
 		float64(3.14),
 		time.Now(),
+		[]string{"Hello", "World"},
+		[]int{42, 21},
+		[2]int{42, 21},
+		[]interface{}{[]int{42, 21}, 42, "Hello", true, []string{"Hello", "World"}},
 	}
-	for n := 0; n < b.N; n++ {
-		for _, value := range values {
-			_ = ToString(value)
-		}
+	for _, value := range values {
+		b.Run(reflect.TypeOf(value).String(), func(b *testing.B) {
```

Comment:
RunParallel should be a separate benchmark funct. That way we can compare between sequential benchmark and concurrent

Reply:
https://github.com/gofiber/utils/pull/63/commits/e00da2ac657e73d44e9822a72e943a24d70cbaac